package.json

```diff
@@ -57,7 +57,8 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "lint-js": "eslint ./ --fix --ext .js --ext .ts --ext .tsx --max-warnings 0 --ignore-pattern node_modules/ --ignore-pattern build/",
+    "generate": "npx graphql-code-generator",
```

Comment:
Question: Will this affect the build or start command?

Reply:
it needs to be run any time a tagged fragment or the big schema file changes for local tooling to work, although all the code that gets generated is .gitignore'd so it doesn't matter for committing code. you can leave `npm run generate -- --watch` running in the background and it will work automagically. 

i wasn't sure what people's dev workflows look like so i didn't want to make opinionated changes to the commands, but we could consider including it as a pre-step in other commands

Reply:
I think if this is needed to run start the application it should be part of the `start` script or maybe if there is a better way to do handle this maybe defined in the README file, but I think ideally we want the same way the application automatically changes with code changes this files should be updated automatically without executing a separate script

Reply:
yeah i agree, i'm just not sure how to do it but i'll fiddle for a bit. the problem is needing to start two background processes, & then making sure that if you stop one you stop the other & that their outputs don't clash. and the most straightforward way is to just start them both in separate terminals manually, even though it's annoying

Reply:
i think i have a better compromise, see https://github.com/techmatters/terraso-web-client/pull/934#issuecomment-1552260099 for details